woocommerce/woocommerce
# Support HTML in checkbox-control tooltip

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Support HTML in checkbox-control tooltip



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Add the following plugin:

```php
<?php

use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;

function YOUR_PREFIX_add_block( BlockInterface $product_name_block ) {
  $parent = $product_name_block->get_parent();

  $parent->add_block(
    array(
      'id' => 'test_id',
      'blockName' => 'woocommerce/product-checkbox-field',
      'attributes' => array(
        'label' => __( 'Test', 'woocommerce' ),
        'property' => 'meta_data.test1',
        'checkedValue' => 'yes',
        'uncheckedValue' => 'no',
        'tooltip' => __( 'Check to exclude this product from the total order quantity and value calculations set up in <a href="/wp-admin/admin.php?page=wc-settings&tab=products" target="_blank">order settings.</a>', 'woocommerce' ),
      )
    )
  );

  $parent->add_block(
    array(
      'id' => 'test_id2',
      'blockName' => 'woocommerce/product-number-field',
      'attributes' => array(
        'label' => __( 'Field', 'woocommerce' ),
        'tooltip' => __( '<a href="https://www.google.com.br">Click me</a>', 'woocommerce' ),
        'property' => 'meta_data.test2',
        )
    )
  );
}

add_action( 'woocommerce_block_template_area_product-form_after_add_block_product-name', 'YOUR_PREFIX_add_block' );

```

2. Enable the new product editor at /wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
3. Go to Products > Add
4. Check the tooltips of the checkbox and the number field added, if the links work correctly.




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
